### PR TITLE
Fixes #24525 - Replace notice with info in UI controller

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: app/controllers/ansible_roles_controller.rb [hunk 1]
```diff
@@ -34,7 +34,7 @@ def import
 
   def confirm_import
     @importer.finish_import(params[:changed])
-    notice _('Import of roles successfully finished.')
+    info _('Import of roles successfully finished.')
```
Comment (bastilian) [inline app/controllers/ansible_roles_controller.rb]: In this instance it might make more sense to use the `success` helper.

**Comment by xprazak2** [comment]
I changed `info` to `success`. `each_value` is also not defined for controller params, so I replaced it with `each`

**Comment by bastilian** [comment]
@xprazak2 cool! However... https://github.com/theforeman/foreman_ansible/pull/186 :) that is also why code climate is not liking it.

**Comment by xprazak2** [comment]
Yeah... I went for a different solution to keep the #186 and make codeclimate happy as well.

**Pull request merged**